admin/basecalendar-admin.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * BaseCalendar admin
+ *
+ * @package WordPress
+ * @subpackage Combined Calendar
+ * @since 1.0.0
+ * @version 1.0.0
+ */
+
+/**
+ * Output_setting_page_html
+ */
+function output_setting_page_html() { ?>
+	<div class="wrap">
```

Comment:
It's all right but I don't prefer this style well... :(
What do you think about the following style?

```php
$html = '<div class="wrap">...';
```